### ISSUE TITLE
Import ES6 modules

### ISSUE THREAD

**Issue by BadIdeaException** [psiho/jsbench-me#69]
Is there a way to import ES6 modules? I can't seem to figure it out. I have tried:

**1. import statement in Setup JavaScript**
```
import { get } from 'https://unpkg.com/object-selectors';
```
but that just results in `TypeError: un is undefined` and an error in console: `Uncaught SyntaxError: import declarations may only appear at top level of a module` . 

**2. dynamic import in Setup JavaScript**
```
const { get } = await import('https://unpkg.com/object-selectors');
```
But it seems that setup code may not be async (`TypeError: un is undefined`, and in console: `Uncaught SyntaxError: await is only valid in async functions, async generators and modules`). 

**3. module script in Setup HTML**
```
<script type="module" src="https://unpkg.com/object-selectors"></script>
```
Again, console shows: `Uncaught SyntaxError: import declarations may only appear at top level of a module`.

And then I ran out of ideas. Is there some way to import an ES module?

**Comment by MirzaAleem** [comment]
1. import statement in Setup JavaScript
`import { get } from 'https://unpkg.com/object-selectors';`
Declarations are always done at the beginning, you must be importing it somewhere else.

2. dynamic import in Setup JavaScript
`const { get } = await import('https://unpkg.com/object-selectors');`
await is only used inside async function.

can you show me the source code, i may help you.

**Comment by BadIdeaException** [comment]
I'm aware of the causes. I listed what I had tried because I was wondering if there was some _other_ (more successful) way of doing this. 

FWIW, I ended up doing my benchmarks [locally](https://benchmarkjs.com/). I'll therefore close this now, but it sure would be nice if importing the subject under test was an option. Just running the Setup Javascript part asynchronously should be enough to at least allow dynamic imports.
